pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -200,14 +195,25 @@ func PrintList(objects []runtime.Object, opt kobject.ConvertOptions) error {
 		}
 		data, err := marshal(convertedList, opt.GenerateJSON)
 		if err != nil {
-			return fmt.Errorf("Error in marshalling the List: %v", err)
+			return fmt.Errorf("error in marshalling the List: %v", err)
 		}
 		printVal, err := transformer.Print("", dirName, "", data, opt.ToStdout, opt.GenerateJSON, f, opt.Provider)
 		if err != nil {
 			return errors.Wrap(err, "transformer.Print failed")
 		}
 		files = append(files, printVal)
 	} else {
+		finalDirName := dirName
+		if opt.CreateChart {
+			finalDirName = dirName + string(os.PathSeparator) + "templates"
+		}
+
+		log.Infof("YAML results in : %s", finalDirName)
```

Comment:
Is this suppose to be debug information?

Reply:
If not, it may be better to indicate that this is where the chart information will be :) For example: `Chart files located at: %s", finalDirName` or something along those lines.

Reply:
@cdrage updated, remove the debug information, thanks very much!